### DISCUSSION TITLE
verdaccio启动服务nginx转发后访问报错

### DISCUSSION THREAD

**Discussion by jackason** [verdaccio/verdaccio#4108] [Q&A]
verdaccio启动服务后本地能正常访问登录，一旦通过nginx转发后访问就会报错 

```
{
  "error": "no such package available"
}
```

用npm login 登录时也会报错

```
npm ERR! code E401
npm ERR! Unable to authenticate, your authentication token seems to be invalid.
npm ERR! To correct this please trying logging in again with:
npm ERR!     npm login
```

请问一下各位这种情况改怎么解决

**Comment by jackason** [comment]
这个是在window服务器上

**Comment by juanpicado** [comment]
You would need to share more information (I also don't have windows so I cannot test anything there) anyway here some examples that can give you some guidance https://github.com/verdaccio/verdaccio/tree/master/docker-examples/v5/reverse_proxy
